operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -233,7 +233,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
   containerNetworkPanel(title, metric, instanceName)::
     $.panel(title) +
     $.queryPanel(
-      'sum by(%(instance)s) (rate(%(metric)s{%(namespace)s,%(instance)s=~"%(instanceName)s"}[$__rate_interval]))' % {
+      'sum by(%(instance)s) (rate(%(metric)s{%(namespace)s,%(instance)s=~".*%(instanceName)s"}[$__rate_interval]))' % {
```

Comment:
The chart allows to override both the `<release_name>-mimir-` part or only the `-mimir-` part.

We can choose to only support the default variant of the current version of the chart: `<release_name>-mimir-<component>`.

WDYT about adding another config option with the helm chart prefix? By default it will be `.*-mimir-` to match the chart default. Otherwise, folks can override it if they've chosen to override the names in the chart. I think this is the direction in which the chart went. I'm slightly against it because it gives too much customizability (google thinks that's not a word). But on the other hand it may be better to keep some parity between the jsonnet and the chart.

Reply:
I 💯  agree with your point. Ideally dashboards should just work out of the box. Make it customizable is a solution (and I'm up to it). The only downside is that if the default config doesn't fit your use case, then you will have to change the config and compile the dashboards (not that it's difficult, it's just a command, but adds friction).

Can `<release_name>` include `-`? If not, then we could have a regex which matches `<word>-<word>-<component>` for the Helm case.

Reply:
I pushed changes to the regex you proposed, waiting on opinion(s) for adding a config option.

Reply:
oh, wow. github again showing its caching functions. I didn't see your comment when I posted mine. and I _am sure_ I refreshed before posting it

Reply:
> Can `<release_name>` include `-`? 

yes, anything that's a valid k8s name is a valid release name